docs/README-EDITING.md

```diff
@@ -1,62 +1,42 @@
-# Handsontable Documentation Editing Guidelines
+# Docs editing guidelines
 
-## Best practices for content
+This page covers guidelines for editing the [Handsontable docs](https://handsontable.com/docs).
 
-Applied to `/next` and `/*.*/`.
+## Maintenance rules
 
-### Mutating API Reference content
+When adding new docs files, check the docs [directory structure](./README.md#handsontable-docs-directory-structure), and follow the guidelines below.
 
-API Reference for the next version is generated automatically, so any changes in `/next/api` will be overwritten. Please avoid changing it.
-
-### JS Doc links
-
-Since we want to support case-insensitive URLs, all links in JsDoc should:
-
-* Before `#`: use `-` to separate words, lower case.
-* After `#`: lower case.
+### Filenames
 
-**For instance:**
+* Use only lower-case characters.
+* To separate words, use `-`.
+* Use the `.md` file extension. The `.vue` extension is disabled.
 
-```js
-// Wrong:
-/** {@link Options#autoColumnSize}. */
+### Front matter
 
-// OK:
-/** {@link options#autocolumnsize Options#autoColumnSize}. */
+Each Markdown file starts with the following front matter tags:
 
+| Tag            | Meaning                                                    | Required |
+|----------------|------------------------------------------------------------|----------|
+| `title`        | The page's header.                                         | Yes      |
+| `permalink`    | The page's **unique** URL.                                 | Yes      |
+| `canonicalUrl` | A canonical URL of the page's latest version.              | Yes      |
+| `metaTitle`    | The page's SEO meta title.                                 | No       |
+| `tags`         | Tags used by the docs search engine.                       | No       |
 
-// Wrong:
-/** {@link HiddenRows} */
+Front matter example:
 
-// OK:
-/** {@link hidden-rows HiddenRows} */
 ```
-
-### Filenames
-
-Using small chars only with `-` as a word separator is recommended.
-
-### Headers
-
-Each markdown file with a content starts by Frontmatter:
-
-```markdown
 ---
 title: Introduction
 permalink: /next/api/
 canonicalUrl: /api/
 ---
-
-# Introduction
 ```
 
-* **title**: A header on a page.
-* **permalink**: Url to this page, **should be unique**.
-* **canonicalUrl**: Canonical url to the latest version of this page.
+### Sitemap front matter
```

Comment:
front matter > Frontmatter (https://vuepress.vuejs.org/guide/frontmatter.html)

Reply:
@krzysztofspilka it's a bit of a mistake on VuePress's part ("front matter" is much more common), but sure, let's keep it consistent with their docs